projects/angular-calendar/src/modules/week/calendar-week-view-event.component.ts

```diff
@@ -30,6 +30,7 @@ import { PlacementArray } from 'positioning';
       <div
         class="cal-event"
         [ngStyle]="{
+          color: weekEvent.event.color?.secondaryForeground,
```

Comment:
Instead of calling it `secondaryForeground`, how about just `text` or `secondaryText`?

I guess it depends on if it would make sense to use it in the month view or not here:
![CleanShot 2022-06-07 at 18 35 32](https://user-images.githubusercontent.com/6425649/172446860-c193204b-5f89-4d49-9c5d-f6a4c37a7a92.png)

I guess that's less of an issue as the background color is consistent each time. WDYT?

Also I think that the AoT compiler will complain, so we'll need to add it to the schema here:

https://github.com/mattlewis92/calendar-utils/blob/master/src/calendar-utils.ts#L32-L35